packages/arcs/src/centers.ts

```diff
@@ -45,6 +45,11 @@ export const useArcCentersTransition = <Datum extends DatumWithArc, ExtraProps =
     mode: ArcTransitionMode = 'innerRadius',
     extra?: TransitionExtra<Datum, ExtraProps>
 ) => {
+    // center label of root node
```

Comment:
This centers the root most label. Not sure if there is a more proper way to handle this, but I don't think it was accounted for originally

![Screen Shot 2021-11-26 at 9 09 56 AM](https://user-images.githubusercontent.com/89929613/143617262-caac73bc-1587-4433-a236-2bd850b3969f.png)
.